chapters/synchronization.txt

```diff
@@ -3999,6 +3985,36 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * The pname:srcAccessMask member of any element of
```

Comment:
any -> each, if I am reading the styleguide right. (applies uniformly).

"any" only in those last four VUs, where we pick only elements with a certain quality.

Reply:
Done!